src/analyzer.rs

```diff
@@ -182,6 +182,44 @@ impl ImportAssertions {
       _ => None,
     }
   }
+
+  pub fn is_known_none(&self, key: &str) -> bool {
+    match self {
+      ImportAssertions::None => true,
+      ImportAssertions::Known(map) => !map.contains_key(key),
+      _ => false,
```

Comment:
Can you change `_` here to be explicit about the variant? That will make the code easier to read without knowing all the variants and cause compile errors here when the variants change. I know we don't do that everywhere in the code here, but we should do it more often.